From: Benjamin Poirier <bpoirier@suse.com>
Date: Tue, 7 May 2019 15:44:50 +0900
Subject: qlge: Remove useless memset
Patch-mainline: v5.5-rc1
Git-commit: 7be4c3f92077e4c5e1423edc5056c890f8bddfc7
References: bsc#1106061

This just repeats what the other memset a few lines above did.

Signed-off-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/qlogic/qlge/qlge_main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/qlogic/qlge/qlge_main.c
+++ b/drivers/net/ethernet/qlogic/qlge/qlge_main.c
@@ -2811,7 +2811,6 @@ static int qlge_init_bq(struct qlge_bq *
 	buf_ptr = bq->base;
 	bq_desc = &bq->queue[0];
 	for (i = 0; i < QLGE_BQ_LEN; i++, buf_ptr++, bq_desc++) {
-		memset(bq_desc, 0, sizeof(*bq_desc));
 		bq_desc->index = i;
 		bq_desc->buf_ptr = buf_ptr;
 	}
